src/content/docs/en/guides/configuring-astro.mdx

```diff
@@ -1,131 +1,163 @@
+<meta property="og:title" content="My New Astro Website" />
+<meta property="og:type" content="website" />
+<meta property="og:url" content="http://www.example.com/" />
+<meta property="og:description" content="Welcome to my new Astro site!" />
+<meta property="og:image" content="https://www.example.com/_astro/seo-banner.BZD7kegZ.webp">
+<meta property="og:image:alt" content="">
+
+<SomeOtherTags />
+
+<Favicon />
 ```
 
-Astro will resolve all other relative file and directory strings as relative to the project root:
+### Reference relative Files
+
+If you provide a relative path to `root` or the `--root` CLI flag, Astro will resolve it against the current working directory where you ran the `astro` CLI command. Astro will resolve all other relative file and directory strings as relative to the project root:
 
 ```js
 // astro.config.mjs
 import { defineConfig } from 'astro/config'
 
 export default defineConfig({
-  // Resolves to the "./foo" directory in your current working directory
-  root: 'foo',
-  // Resolves to the "./foo/public" directory in your current working directory
+  // Resolves to the "./project" directory in your current working directory
+  root: 'project',
+  // Resolves to the "./project/public" directory in your current working directory
   publicDir: 'public',
 })
 ```
 
 To reference a file or directory relative to the configuration file, use `import.meta.url` (unless you are writing a common.js `astro.config.cjs` file).
 
-```js "import.meta.url"
-// astro.config.mjs
+```js title="astro.config.mjs" "import.meta.url"
 import { defineConfig } from 'astro/config'
 
 export default defineConfig({
-  // Resolves to the "./foo" directory, relative to this config file
-  root: new URL("./foo", import.meta.url).toString(),
+  // Resolves to the "./project" directory, relative to this config file
+  root: new URL("./project", import.meta.url).toString(),
   // Resolves to the "./public" directory, relative to this config file
   publicDir: new URL("./public", import.meta.url).toString(),
 })
 ```
 
 :::note 
-Vite-specific `import.meta` properties, like `import.meta.env` or `import.meta.glob`, are _not_ accessible from your configuration file. We recommend alternatives like [dotenv](https://github.com/motdotla/dotenv) or [fast-glob](https://github.com/mrmlnc/fast-glob) for these respective use cases. In addition, [tsconfig path aliases](https://www.typescriptlang.org/tsconfig#paths) will not be resolved. Use relative paths for module imports in this file.
+Vite-specific `import.meta` properties, like `import.meta.env` or `import.meta.glob`, are _not_ accessible from your configuration file. We recommend alternatives like [dotenv](https://github.com/motdotla/dotenv) or [fast-glob](https://github.com/mrmlnc/fast-glob) for these respective use cases.
+
+In addition, [tsconfig path aliases](https://www.typescriptlang.org/tsconfig#paths) will not be resolved. Use relative paths for module imports in this file.
 :::
```

Comment:
This note seems kind of irrelevant to this section? Whether we keep, move, or delete the section, we might want to delete this note.